routes/furniture.js

```diff
@@ -0,0 +1,101 @@
+const express = require('express');
+const { pool, db } = require('../server/db');
+
+const router = express.Router();
+// await pool.query('query')
+
+
+router.get('/', async (req, res) => {
+  try {
+    const furniture = await pool.query('SELECT * FROM furniture');
+    res.status(200).json(furniture.rows);
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+router.post('/', async (req, res) => {
+try {
+    const {
+        id,
+        donation_id,
```

Comment:
🚫 **[eslint]** <[camelcase](https://eslint.org/docs/rules/camelcase)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'donation_id' is not in camel case.

Reply:
All variable names received from the frontend will be in `camelCase` so let's change all of those to address the eslint errors as well.